docs/README.md

```diff
@@ -30,29 +33,81 @@ If you're interested in contributing to the SOK open source project, review the
 
 Review the [Change Log](ChangeLog.md) page for a history of changes in each release.
 
-
 ## Prerequisites for the Splunk Operator
 
-While we are only able to test and support a small subset of configurations, the Splunk Operator should work with any CNCF certified distribution of Kubernetes, version 1.12 or later. Setting up, configuring and managing Kubernetes clusters is outside the scope of this guide and Splunk’s coverage of support. You can submit bugs to https://github.com/splunk/splunk-operator/issues.
+### Supported Kubernetes Versions
+
+- Kubernetes, version 1.16.2+ and later.
+
+The Splunk Operator should work with any [CNCF certified distribution]((https://www.cncf.io/certification/software-conformance/)) of Kubernetes. We do not have platform recommendations, but this is a table of platforms that our developers, customers, and partners have used successfully with the Splunk Operator.
+
+|  |   |
+| ---- | --- |
+| Splunk Development & Testing Platforms | Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE) |
+|Customer Reported Platforms | Microsoft Azure Kubernetes Service (AKS), Red Hat OpenShift |
+ Partner Tested Platforms | HPE Ezmeral|
+| Other Platforms |[CNCF certified distribution](https://www.cncf.io/certification/software-conformance/) |
+|  |   |
+
+
+
+
+
+### Splunk Enterprise Compatibility
+Each Splunk Operator release has specific Splunk Enterprise compatibility requirements. Before installing or upgrading the Splunk Operator, review the [Change Log](ChangeLog.md) to verify version compatibility with Splunk Enterprise releases.
+
+### Splunk Apps Installation
+
+Apps and add-ons can be installed using the Splunk Operator by following the instructions given at [Installing Splunk Apps](Examples#installing-splunk-apps).  Premium apps such as Enterprise Security and IT Service Intelligence are currently not supported.
 
-*Kubernetes releases 1.16.0 and 1.16.1 contain a
-[critical bug(https://github.com/kubernetes/kubernetes/pull/83789) that can
-crash your API server when using custom resource definitions. Do not
-attempt to run the Splunk Operator using these releases. This bug is fixed in
-Kuberenetes 1.16.2.*
 
+### Docker requirements
 The Splunk Operator requires these docker images to be present or available to your Kubernetes cluster:
 
 * `splunk/splunk-operator`: The Splunk Operator image (built by this repository)
```

Comment:
How about "The Splunk Operator image built by this repository or the [official release](https://hub.docker.com/r/splunk/splunk-operator) (1.0.0 RC)"